frontend/src/metabase/static-viz/components/PieChart/legend.tsx

```diff
@@ -5,6 +5,12 @@ import type { PieChartModel } from "metabase/visualizations/echarts/pie/model/ty
 import { calculateLegendRows } from "../Legend/utils";
 import { Legend } from "../Legend";
 
+const FONT = {
```

Comment:
Copied from static row chart since it uses the same underlying legend component

Reply:
We will need to update it once  https://github.com/metabase/metabase/pull/35321/files#diff-bd11417ed8fec62f9f9968f8781d46f1f9af1e92064e1b2926b5216fbaaa07e2R20 gets merged